src/pages/transactions.js

```diff
@@ -1,11 +1,20 @@
+import {useNavigate, useParams} from 'react-router-dom';
+
 import Button from '../components/Button';
+import AccountBalance from '../components/Transaction/AccountBalance';
 import Transactions from '../components/Transaction/Transactions';
 
 export default function TransactionHistory() {
+	const navigate = useNavigate();
+	const accountID = useParams().accountID;
 	return (
```

Comment:
Oh! Okay, I didn't know/think about that since it automatically removes blank lines when I save the files. I'll try to make it more readable once we refactor!

Reply:
Like I said, it is just my personal preference and you can do as you like to 👍 :)